src/App.tsx

```diff
@@ -15,35 +16,116 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  alpha = 'alpha',
+  length = 'length',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: string,
+  reversed: boolean,
+): string[] {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.alpha:
+          return good1.localeCompare(good2);
+        case SortType.length:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reversed) {
+    return preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
```suggestion
  const [sortField, setSortField] = useState(SortType.Default);
```
add the default field to the enum